src/Migration/OldTaskFinder.php

```diff
@@ -36,16 +35,17 @@ public function all(?string $plugin) {
 	 * @return array<string>
 	 */
 	protected function getTasks(string $path, ?string $plugin) {
-		$Folder = new Folder($path);
-		$res = $Folder->find('Queue.+Task\.php');
+		$res = glob($path . '*Task.php');
 
 		$tasks = [];
-		foreach ($res as $key => $r) {
-			$name = basename($r, 'Task.php');
-			$name = substr($name, 5);
-
-			$taskKey = $plugin ? $plugin . '.' . $name : $name;
-			$tasks[$taskKey] = $path . $r;
+		if ($res) {
```

Comment:
instead of nesting, why not just `glob() ?: []` ?

Reply:
The if nesting seems weird

Reply:
I think I get it now